.github/workflows/ci.yml

```diff
@@ -11,18 +11,22 @@ jobs:
       fail-fast: false
       matrix:
         distro:
-        - centos-8
         - centos-9
         - fedora-35
         - fedora-36
+        include:
+          - distro: centos-8
+            env: py36
+          - distro: centos-8
+            env: py38
```

Comment:
I hope we can make this simpler - list of distro/env combinations:

    cs8-py36:
        disto: centos-8
        env: py36
    cs9-py38:
        disto: centos-8
        env: py38
    cs9-py39
        disto: centos-9
        env: py39
    f35-py310:
        disto: fedora-35
        env: py310
    f36-py310:
        disto: fedora-36
        env: py310

Reply:
Lets go with include per distro.

Reply:
Tried this in isolation with the last push, none of the other comments is addressed, but I wanted to see if it executed what we expected (and it does!). I'll cover the rest next week.